docs/README.md

```diff
@@ -0,0 +1,33 @@
+# Docs Directory
+
+## How-Tos
+* [Dev Of The Week Zendesk](dev-of-the-week-zendesk.md) - How to find stuff in zendesk if you are dev of the week
+* [Dropbox Share Guide](dropbox-share-guide.md) - How to add new dropbox shares to staging
+* [Honeybadger](honeybadger.md) - How to use honeybadger to triage site issues
```

Comment:
@joshlory These docs look to be pretty old. Are they still relevant? Have they been superseded by some other docs?